(Content) Remnant News 1 - Generic & Incomprehensible

Zitchas opened the pull request:
**Content (News)**

## Summary
This PR adds a pair of news items:
- "remnant security advisory"
- "remnant incomprehensible"
- "remnant cafeteria"

This PR adds the following phrases:
- "00-07"
- "0-F"
- "00-1F"
- "A group of"
- "Indeterminate"
- "Amount of Remnant"

## Details
This adds spaceport "news" messages to Remnant worlds, and is intended to be baseline messages that are either text advisories that <first> is reading on message boards; or incomprehensible conversations that they are witnessing. These messages are short and to the point. It is expected that an event triggered by Learning Sign 2 will add some "Somewhat comprehensible" messages, and the follow up timer for learning sign would unlock more comprehensible messages, however these are  not included in this PR.

"remnant security advisory" : Is a variety of "departure/arrival board" style notifications intended to convey a sense of activity and threat. For the most part it is statements of things like `System Advisory: Incoming Korath Fleet in 01 days 1B hours. Threat Sierra-2C` or `Reconnaissance Talon D47 standby for launch.` These use quite a few variations to give a large number of possible values for the numbers

"remnant incomprehensible": Is a variety of 'general life' observations by <first>. Basically, stuff they see around the port that they don't understand. Mainly groups signing to each other. ex. `A team of Remnant: They sign precisely to a Remnant at the information desk. You can not understand what they are discussing.`

"remnant cafeteria": Is a variety of "general life" observations that are specific to food service locations.

**Phrases**
With the exception of "Amount of Remnant," none of the phrases are specific to the Remnant. Likewise, they are named precisely what they are to facilitate being used in a sentence in a way that reads appropriately to what they will display.
- "00-07" : random number between 00 and 07, displayed as a two digit number.
- "0-F" : displays a single digit hexadecimal number between 0 and F.
- "00-1F" : displays a two digit hexadecimal number between 00 and 1F.
- "A group of" : displays a term for a group of something in the format "A *term_here* of".
- "Indeterminate" : displays a term for a group that is less defined, such as "A few " or "Lots of "
- "Amount of Remnant" : Creates a sentence using a group term from the previous two phrases and the word "Remnant"

While it is possible that some of these phrases could be consolidated a bit more, I have specifically broken them out the way I have to expose the basic word groups so that others can use these phrases in other places outside of Remnant space.

Note: the `${ }` notation used in the "Incoming Korath Fleet in XX days YY hours" message is an alternate way of calling a phrase. 

## Lore Implications
- The Remnant use hexadecimal in a lot of their easy-to-read numbering.
- The Remnant use a 32 hour clock. Remnant hours may or may not equal Earth hours.

## Save File
Any pilot that can access a Remnant world can test this PR by visiting the spaceport.

## PR Checklist
 - ~~[X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified~~
 - ~~[X] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}~~
 - ~~[X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~~

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,293 @@
+		word
+			"2"
+			"3"
+		word
+			"0"
+			"1"
+			"2"
+			"3"
+			"4"
+			"5"
+			"6"
+			"7"
+			"8"
+			"9"
+			"A"
+			"B"
+			"C"
+			"D"
+			"E"
+			"F"
```
You could remove a lot of duplication by creating a phrase named "0 to F", then referencing that phrase whenever you want 0 to F to show up.
```suggestion
		phrase
			"0 to F"
```
And this would of course happen in multiple other places. The same could be done with the "00 to 07"

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,293 @@
+		word
+			"0"
+			"1"
+		word
+			"0 hours. "
+			"1 hours. "
+			"2 hours. "
+			"3 hours. "
+			"4 hours. "
+			"5 hours. "
+			"6 hours. "
+			"7 hours. "
+			"8 hours. "
+			"9 hours. "
+			"A hours. "
+			"B hours. "
+			"C hours. "
+			"D hours. "
+			"E hours. "
+			"F hours. "
```
Using the phrase "0 to F" idea, this could be greatly condensed.
```suggestion
		phrase
			"0 to F"
		word
			" hours. "
```

Zitchas commented:
Hmmm...  Yeah, I figured there ought to be a way to condense it. So far, my only real exposure to phrases has been the confusing mess that is the Deep/Remnant missions... Not really a fan. But should be fairly simple to use here.

https://github.com/endless-sky/endless-sky/wiki/CreatingPhrases

Just for my own reference.

W1zrad commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,293 @@
+			"C"
+			"D"
+			"E"
+			"F"
```
Probably need to add periods to the end of this phrase.

Zitchas replied:
Done. Thanks!

W1zrad commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,293 @@
+			"C"
+			"D"
+			"E"
+			"F"
```
Same here for periods.

Zitchas replied:
Done. Thanks!

W1zrad commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,293 @@
+			"discussing."
+	message
+		word
+			"They "
```
```suggestion
			"They"
```
Or you can remove spaces from the start of all of the next `word` grouping.

Zitchas commented:
@Amazinite OK, fixed all that duplication. Should be much more streamlined now. 

I realize it is slightly less compact than it could be, but on the upside, the phrases (specifically, the numbers and the group terms) are directly accessible so that others can use them elsewhere.

Also broke the "general life" observations one into general and food specific ones, and expanded the descriptors used. Also changed the "name" used in the news stories for the general scenes to something that sounds more scenery. It flows much better now than it did before.

W1zrad commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+			"another Remnant. "
+			"other Remnant. "
+			"a Remnant at the information desk. "
+			"a dockworker."
```
```suggestion
			"a dockworker. "
```

Amazinite replied:
Better to just have the `. ` be a different word than to forget it in one of these words.

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+
+
+
+phrase "00-07 days"
+	phrase
+		"00-07"
+	word
+		" days"
```
Given that this phrase is just a previous phrase plus one word, it seems pretty pointless to me for it to be its own phrase unless you plan to use it elsewhere/more often. (It's especially redundant when the name of the phrase is just the phrase plus the word that it includes. This can be streamlined by using the following wherever you currently have `phrase "00-07 days"`:)
```
word
	"${00-07 days} days"
```
(This is functionally the exact same thing without having to define a new phrase elsewhere.)

Zitchas replied:
Nice and elegant. I didn't even know that was a thing for ES. I'll be using that, I think. (so long as it keeps the double digits)

Zitchas replied:
OK, I got it to work. For reference if anyone else ever digs through here, `${ }` basically acts like an inline way to call a phrase. So it isn't eliminating the base phrase, just the secondary phrase that combined the first phrase with the word "days".

Thanks!

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+	location
+		government "Remnant"
+	name
+		word
+			"System Security "
+			"Local "
+			"System "
+			"Defense "
+		word
```
```suggestion
		word
			"System security"
			"Local"
			"System"
			"Defense"
		word
			" "	
		word
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+		word
+			"."
+	message
+		word
+			"Patrol "
+			"Salvage "
+			"Sentry "
+			"Attack "
+			"Reconnaissance "
+			"Escort "
+			"Decoy "
+		word
+			"Flight "
+			"Talon "
+			"Wing "
```
```suggestion
		word
			"Patrol"
			"Salvage"
			"Sentry"
			"Attack"
			"Reconnaissance"
			"Escort"
			"Decoy"
		word
			" "
		word
			"Flight"
			"Talon"
			"Wing"
		word
			" "
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+			"0-F"
+		phrase
+			"0-F"
+		word
+			" prepare for "
+			" ready for "
+			" standby for "
+			" hold for "
```
```suggestion
		word
			" "
		word
			"prepare for"
			"ready for"
			"standby for"
			"hold for"
		word
			" "
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+			" ready for "
+			" standby for "
+			" hold for "
+		word
+			"departure."
+			"launch."
+			"combat."
```
```suggestion
		word
			"departure"
			"launch"
			"combat"
		word
			"."
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+			"System "
+			"Defense "
+		word
+			"Advisory"
+			"Status"
+			"Threats"
```
```suggestion
			"advisory"
			"status"
			"threats"
```

Zitchas replied:
This is intended to be in Title Case. Suggesting this has, however, brought to my attention the fact that I missed capitalizing "Security" in the set before it. Thanks!

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+
+
+phrase "00-07"
+	word
+		"00"
+		"01"
+		"03"
+		"04"
+		"05"
+		"06"
+		"07"
```
```suggestion
	word
		"0"
	word
		"0"
		"1"
		"3"
		"4"
		"5"
		"6"
		"7"
```

Zitchas replied:
I intentionally used double digits for stylistic purposes.

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+		word
+			" sign "
+			" gesture "
+		word
+			"quickly "
+			"calmly "
+			"leisurely "
+			"slowly "
+			"deliberately "
+			"carefully "
+			"precisely "
+			"sharply "
+			"bruskly "
+			"briskly "
+			"sweepingly "
+			"intensively "
+			"meticulously "
+			"at length "
+			"vaguely "
+			"briskly "
+			"hurriedly "
+			"hastily "
+			"abruptly "
+			"spontaneously "
+		word
+			"to "
```
```suggestion
		word
			"quickly"
			"calmly"
			"leisurely"
			"slowly"
			"deliberately"
			"carefully"
			"precisely"
			"sharply"
			"briskly"
			"sweepingly"
			"intensively"
			"meticulously"
			"at length"
			"vaguely"
			"briskly"
			"hurriedly"
			"hastily"
			"abruptly"
			"spontaneously"
		word
			" to "
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+			"spontaneously "
+		word
+			"to "
+		word
+			"each other. "
+			"another Remnant. "
+			"other Remnant. "
+			"a Remnant at the information desk. "
+			"a dockworker."
+			"something or someone you can't see. "
```
```suggestion
		word
			"each other"
			"another Remnant"
			"other Remnant"
			"a Remnant at the information desk"
			"a dockworker"
			"something or someone you can't see"
		word
			". "
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+			"something or someone you can't see. "
+		word
+			"You can not "
+		word 
+			"make out "
+			"figure out "
+			"understand "
+			"comprehend "
+		word
+			"what they are "
```
```suggestion
		word 
			"make out"
			"figure out"
			"understand"
			"comprehend"
		word
			" what they are "
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+			"comprehend "
+		word
+			"what they are "
+		word
+			"saying."
+			"communicating."
+			"conveying."
+			"discussing."
```
```suggestion
		word
			"saying"
			"communicating"
			"conveying"
			"discussing"
		word
			"."
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,279 @@
+	message
+		phrase 
+			"Amount of Remnant"
+		word
+			" collect trays heaped with food from the kitchen window."
+			" appear to be enjoying a large meal together."
+			" are relaxing around the remains of a large meal."
+			" are signing to each other over a large meal."
+			" are reading reports over a large meal."
+			" are examining holo projections over a large meal."
```
```suggestion
		word
			" "
		word
			"collect trays heaped with food from the kitchen window"
			"appear to be enjoying a large meal together"
			"are relaxing around the remains of a large meal"
			"are signing to each other over a large meal"
			"are reading reports over a large meal"
			"are examining holo projections over a large meal"
		word
			"."
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,298 @@
+			"Advisory"
+			"Status"
+			"Threats"
+	message
```
Be aware that I have removed the forced quotation marks for news, but news that are quotes should still be surrounded in quotes. You should add
```
word
	`"`
```
to the top and bottom of each of these messages.

Zitchas replied:
Alright, done. Thanks!

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,307 @@
+phrase "A group of"
+	word
+		"A "
+	word
+		"pair "
+		"couple "
+		"pack "
+		"squad "
+		"team "
+		"family "
+		"group "
+		"gathering "
+		"small group "
+		"medium group "
+		"large group "
+	word
+		"of "
```
```suggestion
	word
		"pair"
		"couple"
		"pack"
		"squad"
		"team"
		"family"
		"group"
		"gathering"
		"small group"
		"medium group"
		"large group"
	word
		" of "
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,307 @@
+
+
+phrase "Indeterminate"
+	word
+		"A few "
+		"Several "
+		"Some "
+		"Lots of "
+		"Many "
```
```suggestion
	word
		"A few"
		"Several"
		"Some"
		"Lots of"
		"Many"
	word
		" "
```

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,307 @@
+			"Landing dock"
+			"Spaceport alley"
+			"Spaceport hallway"
+		word
+			" scene"
```
It seems odd to be to describe this as a "scene." If the name of a message isn't a person, then I'd expect it to be a location without it needing to say that.
```suggestion
```

Zitchas replied:
All these are just straight up "what the player sees in the named location."

Amazinite commented on data/remnant/remnant news.txt:
```diff
@@ -0,0 +1,307 @@
+			"Mess"
+			"Canteen"
+			"Kitchen"
+		word
+			" scene"
```
```suggestion
```
